egs/aishell/s10/chain/inference.py

```diff
@@ -28,7 +29,11 @@ def main():
         logging.warning('No GPU detected! Use CPU for inference.')
         device = torch.device('cpu')
     else:
-        device = torch.device('cuda', args.device_id)
+        devices = allocate_gpu_devices(1)
```

Comment:
Should we follow the kaldi style `SelectGpu()` to select a gpu that has the largest available memory?

Reply:
I prefer to do this with `--q` options in `queue.pl`, considering GPUs on a single machine nowdays are commonly with same type (and memory) and we suppose they are in `Exclusive_Process` mode